### PR TITLE
my problems!

### PR DESCRIPTION
**Pull request by theteamaker**

### CODE REVIEW

File: week-01/theteamaker/aboveaverage/1.in [hunk 1]
```diff
@@ -0,0 +1,6 @@
+5
```
Comment (nint8835) [inline week-01/theteamaker/aboveaverage/1.in]: Test files should not be included

**Review by nint8835 — DISMISSED** [review]

File: week-01/theteamaker/aboveaverage/1.in (hunk 1 above)
Comment (theteamaker) [reply week-01/theteamaker/aboveaverage/1.in]: fixed

File: week-02/theteamaker/messybanner.lua [hunk 2]
```diff
@@ -0,0 +1,63 @@
+lines = {}
```
Comment (jackharrhy) [inline week-02/theteamaker/messybanner.lua:1]: image

Comment (jackharrhy) [reply week-02/theteamaker/messybanner.lua:1]: ^ can't run file

**Review by jackharrhy — CHANGES_REQUESTED** [review]
everything works as expected, except for the `messybanner.lua` file not running on my system, maybe give a readme with running instructions?

File: week-02/theteamaker/messybanner.lua (hunk 2 above)
Comment (theteamaker) [reply week-02/theteamaker/messybanner.lua:1]: fixed, I believe

**Review by jackharrhy — APPROVED** [review]

**Pull request merged**